integrations.md

```diff
@@ -199,7 +202,7 @@ packages:
 - title: Background Downloader
   description: (iOS, Android) Simple library download files in a way that allows the download to progress even when the app is in the background.
   url: https://github.com/Anagrr/Fuse.BackgroundDownload
-plugins:
+- Plugins
```

Comment:
Not sure that this is correct; is this not removing the variable reference that's used to produce the plugins list further down below?